### DISCUSSION TITLE
Welcome to domino-mvp Discussions!

### DISCUSSION THREAD

**Discussion by vegegoku** [GwtDomino/domino#52] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by sdhaliwal** [comment]
Hello there,
First of all I want to congratulate the community here for the awesome family of DominoKit projections. I have been reading domino-mvp documentation which is very informative and well-written. However, there are places where some tweaks can make this already excellent documentation even better. I was wondering if there is a way to make updates to some of this documentation solely for the purpose of fixing minor English oversights.

**Reply by vegegoku** [reply]
Thank you for the feedback, I am not sure but I think you can edit the wiki and we can accept the changes.

**Reply by sdhaliwal** [reply]
Ok, thanks. I guess I will clone the wiki repo, make edits and then create a pull request. That is what you meant, right?. I want to make sure we are on the same page :).

**Reply by vegegoku** [reply]
I think this might help
https://docs.github.com/en/github/building-a-strong-community/adding-or-editing-wiki-pages
